composer.json

```diff
@@ -1,32 +1,32 @@
 {
-    "name": "kitpages/data-grid-bundle",
+    "name": "imt/kitpages",
     "type": "symfony-bundle",
-    "description": "Symfony DataGridBundle",
+    "description": "IMT DataGridBundle",
     "keywords": ["Datagrid", "paginator", "filter", "sort"],
-    "homepage": "https://github.com/kitpages/KitpagesDataGridBundle",
+    "homepage": "git@bitbucket.org:ArturTovmasyan/kitpage-new.git",
     "license": "MIT",
     "authors": [
         {
-            "name": "Philippe Le Van",
-            "email": "philippe.levan@kitpages.fr"
+            "name": "Artur Tovmasyan",
+            "email": "artur.tovmasyan777@gmail.com"
         }
     ],
     "require": {
         "php": ">=7.1",
-        "symfony/framework-bundle": "~3.3|~4.0",
-        "symfony/dependency-injection": "~3.3|~4.0",
-        "symfony/translation": "~3.3|~4.0",
-        "symfony/templating": "~3.3|~4.0",
+        "symfony/framework-bundle": "~4.4|~5.4",
```

Comment:
I don't try the Symfony 6 version, because I use PHP 7.4, for SF6 version PHP 8 version is required. But I believe that the SF6 version can be used without any risks.